sdk/python/tests/integration/feature_repos/universal/feature_views.py

```diff
@@ -39,7 +39,7 @@ def create_customer_daily_profile_feature_view(source):
             Feature(name="current_balance", dtype=ValueType.FLOAT),
             Feature(name="avg_passenger_count", dtype=ValueType.FLOAT),
             Feature(name="lifetime_trip_count", dtype=ValueType.INT32),
-            Feature(name="avg_daily_trips", dtype=ValueType.INT32),
+            # Feature(name="avg_daily_trips", dtype=ValueType.INT32),
```

Comment:
intentional comment?

Reply:
Yes, this feature doesnt actually exist in the data backed by this view.

Reply:
can we just remove it then?